packages/neuron-ui/src/containers/MainContent/reducer.ts

```diff
@@ -122,6 +150,7 @@ export const actionCreators = {
       payload: ns,
     }
   },
+
   deleteNetwork: (name: string) => {
     if (name === Testnet) {
```

Comment:
Perhaps better have an internal flag or identifier. (Just in case we remove the unique name requirements, under which name comparison is insufficient.